framework/i18n/Formatter.php

```diff
@@ -578,16 +577,18 @@ public function asImage($value, $options = [])
      * Formats the value as a hyperlink.
      * @param mixed $value the value to be formatted.
      * @param array $options the tag options in terms of name-value pairs. See [[Html::a()]].
+     * @param bool $secureProtocol whether to use secure version of HTTP protocol when URL does not contain
+     * the protocol part (available since 2.0.43).
      * @return string the formatted result.
      */
-    public function asUrl($value, $options = [])
+    public function asUrl($value, $options = [], $secureProtocol = false)
```

Comment:
How about making it consistent with `Url::toRoute($route, $scheme = false)`, `Url::to($url = '', $scheme = false)`, etc.

Reply:
I've tried to make it as consistent as possible without adding extra complexity with additional dependency on UrlManager while keeping it BC.